.travis.yml

```diff
@@ -48,6 +48,11 @@ install:
   # libimagequant
   - pushd depends && ./install_imagequant.sh && popd
 
+before_script:
+# QT needs a display for some of the tests, and it's only run on the system site packages install
```

Comment:
No. It's not needed. However. It takes .01 seconds to start, (https://travis-ci.org/python-pillow/Pillow/jobs/171335292#L1933)  and that's far less time than I spent faffing around to try to get the yml syntax right.